src/xenialdan/MagicWE2/Loader.php

```diff
@@ -5,10 +5,15 @@
 namespace xenialdan\MagicWE2;
 
 use pocketmine\lang\Language;
```

Comment:
except your still using 3.0.0 path, lol

Reply:
In the src code of the mcpe-1.8 branch that i use this is already Language